packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.ts

```diff
@@ -2738,6 +2738,16 @@ export class SavedObjectsRepository implements ISavedObjectsRepository {
     });
   }
 
+  /**
+   * {@inheritDoc ISavedObjectsRepository.getCurrentNamespace}
+   */
+  getCurrentNamespace(namespace?: string) {
```

Comment:
__note:__ I had to expose this method from the repository and the client to get the current namespace in the import code. The current namespace is required to properly generate legacy URL alias ID.

Reply:
> Do we really need to mirror the API definition of SpaceExtension.getCurrentNamespace? Or would it be acceptable to just have a SOR.getCurrentNamespace(-no-parameters-)?

That's what I would like to explore further, I agree this `namespace` argument is super confusing. If I cannot find any use case for that parameter, I'll happily get rid of it.

Reply:
> Would there ever be a scenario where spaces as a feature is enabled but the spaces SO extension is not?

I don't think it can be the case (otherwise having or not having the weird `namespace` parameter in `getCurrentNamespace` method would be the least of our problems :slightly_smiling_face: )

> As the client doesn't include the parameter, and it is the only external consumer of the new repo method. 

I removed it from the client's public method signature, but kept in the repository's signature since it's still used internally.